podcasts/Sharing/SharingView.swift

```diff
@@ -2,36 +2,94 @@ import SwiftUI
 import PocketCastsDataModel
 import PocketCastsUtils
 
+class ClipTime: ObservableObject {
```

Comment:
This isn't really used for much right now but will be used as clip trimming is added.